tools/export-neptune-tool/src/main/scala/cmwell/tools/neptune/export/ExportToNeptuneManager.scala

```diff
@@ -113,10 +113,12 @@ class ExportToNeptuneManager(ingestConnectionPoolSize: Int) {
     res
   }
 
-  def persistDataInS3AndIngestToNeptuneViaLoaderAPI(neptuneCluster: String,bulkResponseAsString:String, nextPosition: String, updateMode: Boolean, readInputStreamDuration: Long, totalInfotons: String, proxyHost:Option[String], proxyPort:Option[Int]) = {
+  def persistDataInS3AndIngestToNeptuneViaLoaderAPI(neptuneCluster: String, bulkResponseAsString:String, nextPosition: String, updateMode: Boolean, readInputStreamDuration: Long, totalInfotons: String, proxyHost:Option[String], proxyPort:Option[Int]) = {
       val startTimeMillis = System.currentTimeMillis()
       val fileName = "cm-well-file-" + startTimeMillis + ".nq"
-      S3ObjectUploader.persistChunkToS3Bucket(bulkResponseAsString, fileName, proxyHost, proxyPort)
+      val allQuads = bulkResponseAsString.split("\n")
+      val bulkResWithoutMeta = allQuads.filterNot(quad => if(quad!= "") quad.split(" ")(1).contains("meta/sys") else false).mkString("\n")
```

Comment:
will be better to do it like so: `.filterNot(_.contains("cmwell://meta/sys"))`

1. no need to `if ... else false` 
2. no need to split - testing for "meta/sys" prefixed with the special "cmwell" protocol is enough, and we are boosting performance when avoiding `split`.
3. why is it important to filter out empty lines? if it's critical, you can do that by `.filterNot(q => q.isEmpty || q.contains("cmwell://meta/sys")`